lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -115,26 +123,42 @@ $-separator-color: var(--nx-swatch-indigo-40);
     margin: 0;
   }
 
-  &:focus {
-    border-radius: 0;
-    outline: 2px solid $-focus-color;
-    outline-offset: -2px;
-  }
+  &.nx-text-link {
+    &:focus {
+      border-radius: 0;
+      outline: var(--nx-focus-outline-width) solid var(--nx-color-global-sidebar-border-focus);
+      outline-offset: calc(0px - var(--nx-focus-outline-width));
+    }
 
-  &:hover {
-    background-color: $-hover-color;
-  }
+    &:focus:hover, &.selected:focus {
```

Comment:
This could be `&:hover, &.selected` inside the `:focus` block above.

Reply:
> This could be `&:hover, &.selected` inside the `:focus` block above.

7629b31bbbbff00d2d86c373f0c99fe5fb27c78f